File: Competition.php
<?php

namespace App\Entity;

use App\Repository\CompetitionRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=CompetitionRepository::class)
 */
class Competition
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     * @Assert\NotBlank(message="le nom est vide")
     */
    private $Nom;

    /**
     * @ORM\Column(type="text", nullable=true)
     *  @Assert\NotBlank(message="description est vide")
     */
    private $description;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     *  @Assert\NotBlank(message="Categorie est vide")
     */
    private $category;

    /**
     * @ORM\Column(type="integer", nullable=true)
     *  @Assert\NotBlank(message="nombre de place est vide")
     */
    private $nombre_de_place;

    /**
     * @ORM\Column(type="text", nullable=true)
     *  @Assert\NotBlank(message="image est vide")
     */
    private $image;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     *  @Assert\NotBlank(message="adresse est vide")
     */
    private $adresse;

    /**
     * @ORM\Column(type="date", nullable=true)
     *  @Assert\NotBlank(message="la date est vide")
     */
    private $date;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->Nom;
    }

    public function setNom(?string $Nom): self
    {
        $this->Nom = $Nom;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(?string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getCategory(): ?string
    {
        return $this->category;
    }

    public function setCategory(?string $category): self
    {
        $this->category = $category;

        return $this;
    }

    public function getNombreDePlace(): ?int
    {
        return $this->nombre_de_place;
    }

    public function setNombreDePlace(?int $nombre_de_place): self
    {
        $this->nombre_de_place = $nombre_de_place;

        return $this;
    }

    public function getImage(): ?string
    {
        return $this->image;
    }

    public function setImage(?string $image): self
    {
        $this->image = $image;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(?string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(?\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }
    public function __toString()
    {
        return (string) $this->getId();
    }
}

File: CompCrud/src/esprit/compcrud/competition.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package esprit.compcrud;

/**
 *
 * @author Maya
 */
public class competition {
    private Integer id;
     private String nom;
      private String description;
      private String categorie ;
       private Integer nombredeplace;
       private String image;
       private String adresse;
       private Integer date ;

    public competition(Integer id, String nom, String description, String categorie, Integer nombredeplace, String image, String adresse, Integer date) {
        this.id = id;
        this.nom = nom;
        this.description = description;
        this.categorie = categorie;
        this.nombredeplace = nombredeplace;
        this.image = image;
        this.adresse = adresse;
        this.date = date;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getCategorie() {
        return categorie;
    }

    public void setCategorie(String categorie) {
        this.categorie = categorie;
    }

    public Integer getNombredeplace() {
        return nombredeplace;
    }

    public void setNombredeplace(Integer nombredeplace) {
        this.nombredeplace = nombredeplace;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public Integer getDate() {
        return date;
    }

    public void setDate(Integer date) {
        this.date = date;
    }

    @Override
    public String toString() {
        return "competition{" + "id=" + id + ", nom=" + nom + ", description=" + description + ", categorie=" + categorie + ", nombredeplace=" + nombredeplace + ", image=" + image + ", adresse=" + adresse + ", date=" + date + '}';
    }

}
